docs/manual/comparison_with_rdatatable.rst

```diff
@@ -0,0 +1,330 @@
+
+``DT[, tail(.SD,2), by=x]``                       ``df[-2:, :, by("x")]``                          Last 2 rows of each group
+========================================         ============================================     ============================================================
+
+In `Rdatatable <https://rdatatable.gitlab.io/data.table/index.html>`_, the order of the groupings is preserved; in ``datatable``, the returned dataframe is sorted on the grouping column. ``DT[, sum(v), keyby=x]`` in Rdatatable returns a dataframe ordered by column ``x``.
+
+Also, in ``datatable``, :ref:`f-expressions` in the ``i`` section of a groupby is not yet implemented, hence the chaining method to get the sum of column ``v`` where ``x!=a``.
+
+Multiple aggregations within a group can be executed in `Rdatatable <https://rdatatable.gitlab.io/data.table/index.html>`_ with the syntax below ::
+
+    DT[, list(MySum=sum(v),
+              MyMin=min(v),
+              MyMax=max(v)),
+       by=.(x, y%%2)]
+
+The same can be replicated in ``datatable`` by using a dictionary ::
+
+    df[:, {MySum: dt.sum(f.v),
+           MyMin: dt.min(f.v),
+           MyMax: dt.max(f.v)},
+       by(f.x, f.y%2)]
+
+
+Add/Update/Delete Column
+-------------------------
+
+========================================         ============================================     ============================================================
+RDatatable                                           Datatable                                              Action
+========================================         ============================================     ============================================================
+``DT[, z:=42L]``                                 | ``df[:, update(z=42)]`` or                       Add new column
+                                                 | ``df['z'] = 42`` or
+                                                 | ``df[:, 'z'] = 42`` or
+                                                 | ``df = df[:, f[:].extend({"z":42})]``
+``DT[, z:=NULL]``                                | ``del df['z']`` or                               Remove column
+                                                 | ``del df[:, 'z']`` or
+                                                 | ``df = df[:,f[:].remove(f.z)]``
+``DT["a", v:=42L, on="x"]``                      | ``df[f.x=="a", update(v=42)]`` or                Subassign to exiting ``v`` column
+                                                 | ``df[f.x=="a", 'v'] = 42``
+``DT["b", v2:=84L, on="x"]``                     | ``df[f.x=="b", update(v2=84)]`` or               Subassign to new column (NA padded)
+                                                 | ``df[f.x=='b', 'v2'] = 84``
+``DT[, m:=mean(v), by=x]``                       | ``df[:, update(m=dt.mean(f.v)), by("x")]``       Add new column by group
```

Comment:
"Add new column by group" -> "Add new column assigning values group-wise"?

Reply:
I'll take your suggestion, since it is more explicit